compiler/rustc_middle/src/hir/mod.rs

```diff
@@ -35,7 +35,7 @@ pub struct IndexedHir<'hir> {
 /// `HirId::local_id == 0`, and excludes bodies.
 #[derive(Copy, Clone, Debug)]
 pub struct Owner<'tcx> {
-    node: Node<'tcx>,
+    node: OwnerNode<'tcx>,
 }
```

Comment:
Does it still make sense to introduce the `Owner` structure?
`OwnerNode` has pretty much the same meaning now.

Reply:
It is used to encapsulate access outside of rustc_middle::hir and to implement HashStable.

Reply:
Could you add this as a comment?